test/Greentube.Monitoring.Tests/ResourceMonitorTests.cs

```diff
@@ -24,7 +24,7 @@ private sealed class Fixture
                 TimeSpan.FromMilliseconds(0),
                 TimeSpan.FromSeconds(2));
             public ILogger<ResourceMonitor> Logger { get; set; } = Substitute.For<ILogger<ResourceMonitor>>();
-            private bool IsCritical { get; } = true;
+            public  bool IsCritical { get; set;  } = true;

```

Comment:
why changed? don't see necessity to assign property.
Assigning directly in tests is not the argument but smell of leaking abstraction

Tests should not be reason to change access modifier. Even Internal and InternalsVisibleTo is generally bad practice

Reply:
Fixture.isCritical is a helper sugar to create ResourceMonitor, so it's ok to make it public. 

Otherwise it will be 
return new ResourceMonitor(
                    ResourceName,
                    HealthCheckStrategy,
                    ResourceMonitorConfiguration,
                    Logger,
                    **True**,
                    TimerFactory);
instead of just GetSut(), it's silly.

Reply:
Yep, initial implementation is very ugly